src/reportWebVitals.js

```diff
@@ -0,0 +1,15 @@
+const reportWebVitals = (onPerfEntry) => {
+  if (onPerfEntry && onPerfEntry instanceof Function) {
+    import('web-vitals').then(({
+      getCLS, getFID, getFCP, getLCP, getTTFB,
+    }) => {
+      getCLS(onPerfEntry);
+      getFID(onPerfEntry);
+      getFCP(onPerfEntry);
+      getLCP(onPerfEntry);
+      getTTFB(onPerfEntry);
+    });
+  }
+};
+
+export default reportWebVitals;
```

Comment:
- [ ] According to the requirements, You should delete all the boilerplate that got created by the CRA (texts, images, and styles), so Kindly remove these files/images from your project ⬇

![1](https://user-images.githubusercontent.com/45231276/204942114-83f1bf02-4fb5-46bf-9210-4ef4c39479a6.PNG)